### PR TITLE
Update configuration-cookbook.md

### PR DESCRIPTION
**Pull request by thomasr8**
Added content and recipes for AWS-EKS Kubernetes.

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/configuration-cookbook.md [hunk 1]
```diff
@@ -462,3 +952,34 @@ version: 2.1
 Notice in the example that the job is run and a Slack status alert is sent to your recipients (USERID1, USERID2) if the job has failed.
 
 For more detailed information about this orb and its functionality, refer to the Slack orb in the [CircleCI Orb Registry](https://circleci.com/orbs/registry/orb/circleci/slack).
+
+## Dependency Caching Strategies in Yarn, Bundler, and Other Popular Package Managers
```
Comment (michelle-luna) [inline jekyll/_cci2/configuration-cookbook.md]: I think you want to remove these empty sections from this PR

File: jekyll/_cci2/configuration-cookbook.md [hunk 2]
```diff
@@ -62,8 +62,12 @@ Configuration Recipe | Description
 ------------|-----------
 Deploying Software Changes to Amazon Elastic Container Service (ECS) | This section describes how you can deploy changes to the Amazon Elastic Container Service (ECS) using a CircleCI-certified ECS orb.
 Deploying Software Changes to Google Kubernetes Engine (GKE) | This section describes how you can deploy changes to the Google Kubernetes Engine (GKE) using a CircleCI-certified GKE orb.
+Using Amazon Elastic Container Service for Kubernetes (Amazon EKS) | This section describes how you can use the Amazon ECS service for Kubernetes for Kubernetes-related tasks and operations.
 Deploying Applications to Heroku | This section describes how you can deploy application to the Heroku platform using the CircleCI Heroku orb.
 Enabling Custom Slack Notifications in CircleCI Jobs | This section describes how you can enable customized Slack notifications in CircleCI jobs.
+Enabling Single-threading Builds on a Branch |
```
Comment (michelle-luna) [inline jekyll/_cci2/configuration-cookbook.md]: Remove these three lines that will be added in the future

File: jekyll/_cci2/configuration-cookbook.md (hunk 1 above)
Comment (thomasr8) [reply jekyll/_cci2/configuration-cookbook.md]: fixed

File: jekyll/_cci2/configuration-cookbook.md (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/configuration-cookbook.md]: fixed

**Pull request merged**